src/dagmc/tests/dagmc_simple_test.cpp

```diff
@@ -29,26 +29,24 @@ TEST_F(DagmcSimpleTest, dagmc_load_file) {
 TEST_F(DagmcSimpleTest, dagmc_load_file_dagmc) {
   /* 1 - Test with external moab, load file in DAGMC*/
   // make new moab core
-  Core* mbi = new moab::Core();
+  std::shared_ptr<Core> mbi = std::shared_ptr<Core>(new moab::Core());
```

Comment:
```suggestion
  std::shared_ptr<Interface> mbi = std::make_shared<Core>();
```

Reply:
I am ok with that suggestion. I just like to understand why sometimes do we use `Core` and some other time `Instance` ?

Reply:
We talked about this offline, but I'll put it here for future reference. 

This works b/c `moab::Core` inherits from `moab::Interface`. The compiler is ok with us assigning the `shared_ptr` of a child class to a `shared_ptr` of a parent class, but this would not work the other way around.